doc/network-ng.md

```diff
@@ -38,31 +40,77 @@ other classes.
   Currently it is bound to an interface name, but we plan to provide more advanced matchers.
 * {Y2Network::Routing}: holds routing information, including IP forwarding settings, routing tables, etc.
 
-## Backend Support
+### Multi-Backend Support
+
+As mentioned above, Y2Network is designed to support different backends. It is expected to implement
+a reader and a writer for each backend (except for AutoYaST, which is an special case). The reader
+will be responsible for checking the system's configuration and building a {Y2Network::Config}
+object, containing interfaces, configurations, routes, etc. On the other hand, the writer will be
+responsible for updating the system using that configuration object.
+
+As a developer, you rarely will need to access to readers/writers because `Yast::Lan` already offers
+an API to read and write the configuration. See the [Accessing the
+Configuration](#accessing-the-configuration) section for further details.
 
-In order to support a new backend, we need to implement a configuration readers and writers. The
-{Y2Network::Sysconfig} module implements support to deal with `sysconfig` files.
+#### Sysconfig
+
+The sysconfig backend support is composed by these files:
 
     src/lib/y2network/sysconfig
     ├── config_reader.rb <- READER
     ├── config_writer.rb <- WRITER
-    ├── connection_config_reader_handlers
-    │   ├── eth.rb
-    │   ├── wlan.rb
-    │   └── ...
     ├── connection_config_reader.rb
+    ├── connection_config_readers
+    │   ├── eth.rb
```

Comment:
just curios is it eth or according to new types ethernet? just that we should be consistent.